doc/developer/bestPractices/ContributorInfo.txt

```diff
@@ -13,6 +13,16 @@ account, signing the contributor agreement, and submitting pull requests.
 Developer Workflow
 ------------------
 
+Overview:
+
+#. `Fork repo on github`_
+#. `Create new branch`_
+#. `Submit pull request`_
+#. `Test changes`_ (can be done by team at Cray)
+#. `Mail chapel-developers@ for review/merge`_
+
+.. _Fork repo on github:
```

Comment:
Note that, whether intentionally or not, these anchors aren't rendered in the .html version of this file you pointed me to.  I think one would want to see the steps corresponding to the five items in the summary in the body of the documentation as well, in part to have a visual anchor to relate back to the quick summary; and in part to break up what is otherwise a pretty large file.

Main downside to doing so that I can see is that it makes reading the file as a .txt file a little less attractive.  But if we changed our www developer resources page to point to the file on GitHub the number of people who read it as a text file will go down significantly (and as all the docs/ content starts getting reflected at chapel.cray.com, it'll matter less and less).